packages/device_info_plus/device_info_plus/example/lib/main.dart

```diff
@@ -21,7 +21,7 @@ void main() {
 }
 
 class MyApp extends StatefulWidget {
-  const MyApp({Key? key}) : super(key: key);
+  const MyApp({super.key});
```

Comment:
The changes in device_info_plus are already being handled in https://github.com/fluttercommunity/plus_plugins/pull/2688

This change should be reverted in this PR